ui/main_window.cpp

```diff
@@ -896,6 +906,11 @@ void MainWindow::CreateActions()
     m_about_action = new QAction(tr("&About Dive"), this);
     m_about_action->setStatusTip(tr("Display application version information"));
     connect(m_about_action, &QAction::triggered, this, &MainWindow::OnAbout);
+
+    // Shortcuts action
+    m_shortcuts_action = new QAction(tr("&Shortcuts"), this);
+    m_shortcuts_action->setStatusTip(tr("Display application keyboard shortcuts"));
+    connect(m_shortcuts_action, &QAction::triggered, this, &MainWindow::OnShortcuts);
```

Comment:
A bit nitpicky: Generally the "About" menu item is last on the Help menu. Maybe we should follow that convention as well?